.github/workflows/checks.yml

```
@@ -7,25 +7,32 @@ on:
       - '**'
   push:
     branches:
-      - 'main'
+      - 'v0'
 
 jobs:
   build_all:
     name: Build All
-    runs-on: ubuntu-22.04
+    runs-on:
```

```suggestion
    runs-on:
            # https://namespace.so/docs/features/faster-github-actions#using-runner-labels
```

Nit: I seem to have missed this link the in -deployment repo, but I always need it.